### PR TITLE
update according to new autoscaling strategy

### PR DESCRIPTION
**Pull request by ErikKaum**
- docs according to this update: https://github.com/huggingface/hf-endpoints/pull/1567

### CODE REVIEW

**Review by beurkinger — APPROVED** [review]
👍

File: docs/source/autoscaling.mdx [hunk 1]
```diff
@@ -12,6 +12,14 @@ The autoscaling process is triggered based on the accelerator's utilization metr
 
 It's important to note that the scaling up process takes place every minute and scaling down takes place every 2 minutes. This frequency ensures a balance between responsiveness and stability of the autoscaling system, with a stabilization of 300 seconds once scaled down. 
 
+### Scaling based on pending requests (experimental feature)
```
Comment (Michellehbn) [inline docs/source/autoscaling.mdx:15]:
```suggestion
### Scaling based on pending requests (beta feature)
```

**Review by Michellehbn — APPROVED** [review]
ty!!

File: docs/source/autoscaling.mdx [hunk 2]
```diff
@@ -12,6 +12,14 @@ The autoscaling process is triggered based on the accelerator's utilization metr
 
 It's important to note that the scaling up process takes place every minute and scaling down takes place every 2 minutes. This frequency ensures a balance between responsiveness and stability of the autoscaling system, with a stabilization of 300 seconds once scaled down. 
 
+### Scaling based on pending requests (beta feature)
+
+You can change the scaling criteria to be based on pending requests instead of utilization metrics. This is currently an experimental feature, so it may change, and we don't recommend using it for production workloads.
+
+- pending requests are requests that have not yet received an HTTP status, meaning they include in-flight requests and requests currently being processed.
+- if there are more than 1.5 pending requests per replica in the past 20 seconds, it triggers an autoscaling event and adds a replica to your deployment.
```
Comment (oOraph) [inline docs/source/autoscaling.mdx:20]: I would also specify that the default threshold (1.5) can be changed too no ?

Comment (ErikKaum) [reply docs/source/autoscaling.mdx:20]: actually in this iteration it's not exposed to the user to be configured, but that will change asap

**Review by oOraph — APPROVED** [review]

**Pull request merged**